launch.go

```diff
@@ -171,6 +195,24 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath string) error
 		return fmt.Errorf("fetching Pipeline ID %d: %v", j.PipelineID, err)
 	}
 
+	log.Printf("Fetching Parent Build %d for Meta", b.ParentBuildID)
```

Comment:
Thank you for show examples! 
I think the value which is defined but not initialize in Go is set to `0`. `b.ParentBuildiD` is maybe 0 when the build does not have `prentBuildId`. The skip condition should be 0.